tests/unit/calendar/events.js

```diff
@@ -0,0 +1,104 @@
+define( [
+	"jquery",
+	"./helper",
+	"ui/widgets/calendar"
+], function( $, testHelper ) {
+
+var element;
+
+module( "calendar: events", {
+	setup: function() {
+		element = $( "#calendar" ).calendar();
+	}
+} );
+
+test( "change", function() {
+	expect( 6 );
```

Comment:
This is using globals. Should use `assert.expect()` (`assert` gets passed as a parameter to the `test()` callback).

Looks like the datepicker branch is missing some updates to not allow the use of the QUnit globals. I guess we'll address that separately.